client/my-sites/plan-features/index.jsx

```diff
@@ -113,6 +113,9 @@ export class PlanFeatures extends Component {
 							initialSelectedIndex={ initialSelectedIndex }
 						>
 							<table className={ tableClasses }>
+								<caption class="screen-reader-text">
```

Comment:
@lcollette FYI, this line is triggering warning in Console:

<img width="580" alt="screenshot_825" src="https://user-images.githubusercontent.com/127594/62321708-1e4d1280-b458-11e9-8b6e-5c1c00b0f580.png">

Replacing `class=` with `className=` cleared the warning.